protocol/validation/merkle_test.go

```diff
@@ -16,31 +15,31 @@ func TestCalcMerkleRoot(t *testing.T) {
 	}{{
 		witnesses: [][][]byte{
 			[][]byte{
-				vm.Int64Bytes(1),
+				{1, 0, 0, 0, 0, 0, 0, 0},
```

Comment:
This works, but it isn't the same as `vm.Int64Bytes(1)` - that's simply `[]byte{1}`.